src/D3MHub.sol

```diff
@@ -331,6 +333,16 @@ contract D3MHub {
         emit Unwind(ilk, amount, fees);
     }
 
+    function _fix(bytes32 ilk, address _pool) internal returns (uint256 diff) {
```

Comment:
It's the same net effect in a bit more optimized way. If you apply `fix` first it will increase the debt and the surplus. Which will be balanced out when grabbing the position.

Reply:
I just deleted my prev comment as I think you might be right, need to think a bit in the scenarios and will get back.

Reply:
I actually think we don't have to use `fix` in `cull` at all but also removing the actual rebalancing from it.
We originally stated that if `cull` happens we should only take the real actual debt as the `gem` to keep that reference. However if someone donated DAI paying back this CDP in reality is donating DAI to the surplus buffer, so maybe the concept of leaving the reference to the actual art is actually wrong, it should always been the actual `ink`.

If we remove the actual rebalancing, calling `fix` or not is the exact same net result, as whatever we send to the surplus buffer in `fix` will be balanced out when the `cull` `grab` is called.